tests/trainer/logging_tests/test_eval_loop_logging_1_0.py

```diff
@@ -541,16 +539,14 @@ def validation_step(self, batch, batch_idx):
     trainer.fit(model)
     trainer.test()
 
-    """
     assert test_callback.funcs_called_count["on_epoch_start"] == 1
-    assert test_callback.funcs_called_count["on_batch_start"] == 1
+    # assert test_callback.funcs_called_count["on_batch_start"] == 1
```

Comment:
Should we remove these completely rather than comment them out? or are they explaining something?

Reply:
Pretty to keep it if batch_start is re-actived in the future.